docs/blog/moving-portland-prague-2020-online.rst

```diff
@@ -19,9 +19,9 @@ Your Portland conference ticket
 
 If you currently have a ticket to Write the Docs Portland, your options are:
 
-- Pre-order a ticket to the online event for $100 and get the rest of your original ticket amount refunded.
-- Pre-order a ticket to the online event for $100 and contribute the rest of your original ticket amount to the Write the Docs organization. We are a self-sustaining community and our events are our main source of income, so your contributions help ensure that we can continue to organize events and maintain the global community in years to come.
-- Get a full refund of your original ticket amount on **May 10th, 2020**. This option will happen automatically if you don't select either of the previous options.
+- Pre-order a ticket to the online event for 25% and get the rest of your original ticket amount refunded.
+- Pre-order a ticket for all of our online events until the end of 2021. We plan to move Portland & Prague online this year at least, and offer similar online events next year. This helps support the Write the Docs organization by helping us cover operating expenses for the next year and a half.
```

Comment:
Would it make sense to say something like 

> Transfer your existing ticket into a virtual ticket pass, valid for all of our online events until the end of 2021.

And I'd delete the "This helps..."